landing_page/app/models/ability.rb

```diff
@@ -4,9 +4,16 @@ class Ability
   def initialize(user)
 
     can :view, User do |u|
-       user.present? && user.id == u.id  
+       user.present? && user.id == u.id
     end
 
+    can :update, User do |u|
+      user.present? && user.id == u.id
+    end
+
+    can :destroy, User do |u|
+      user.present? && user.id == u.id
+    end
```

Comment:
How can we remove this duplication?

Reply:
I think you might be able to do something like:
```ruby
can [:view, :update, :destroy], User do |u|
    user.present? && user.id == u.id
end
```